Gopkg.lock

```diff
@@ -76,7 +82,8 @@
 [[projects]]
   name = "github.com/sergi/go-diff"
   packages = ["diffmatchpatch"]
-  revision = "da645544ed44df016359bd4c0e3dc60ee3a0da43"
+  revision = "1744e2970ca51c86172c8190fadad617561ed6e7"
```

Comment:
This breaks the algorithm. The hash must be exactly that - it contains my fix for the levenshtein distance